CartMoveController.py

```diff
@@ -0,0 +1,90 @@
+import RPi.GPIO as GPIO
+import time
+
+class CartMoveController(object):
+    def __init__(self, speed):
+        self.speed = speed
```

Comment:
Units would be nice to help users understand what they must provide the constructor.